thunder_print.module

```diff
@@ -77,3 +78,34 @@ function thunder_print_entity_operation_alter(array &$operations, EntityInterfac
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_views_view_field().
+ */
+function thunder_print_preprocess_views_view_field(&$variables) {
+
+  /** @var \Drupal\views\ViewExecutable $view */
+  $view = $variables['view'];
+  if ($view->id() == 'thunder_print_snippet_templates') {
```

Comment:
This should work in general and not only on the single view. So looking for the config entity type on the field might be more generic.  `entity_type: print_article_type`